### ISSUE TITLE
A note regarding temporary solution for aliyun

### ISSUE THREAD

**Issue by esquaredly** [circlingchina/circling#68]
The migration of circling to Aliyun has a lot of complications, and it is not something I can do in 3 days (I estimate it would take a full time engineer 2 to 4 weeks). So in the mean time I have deployed a non-functional site (call it circling_cn). So the current setup is:

1) on aliyun server(47.95.8.171), we are running circling_cn, which is missing HTTPS and login. It also can only be updated manually.
2) on netlify(circlingchina.netlify.app) runs the actual website.

http://circlingquanquan.com will point to the fake server, which hopefully satisfy ICP licensing requirements. 

Real users can use circlingchina.netlify.app to access the website

---

What needs to be done to get the real website working on aliyun:
- https
- a new login system, to replace netlify identity. After the new login system is built, data needs to be migrated
- specific adaptations to aliyun. For example, the current docker registry is not accessible on aliyun (blocked by GFW)
- automated build pipeline

(all these things we get for FREE on netlify for $10/month, but it takes hundreds of hours to replicate them on aliyun where these tools are not available)

---

In short I don't think it's possible for me to support a version of circling running on aliyun. Circling started as a static website, but has grown into a fully blown software project with User systems, databases, APIs, payments, and stuff. The only way 1-2 partime volunteers can support such a website is by creatively leveraging many existing tools out there. In my opinion, the only way to maintain such a site in China, where fundamental tools are lacking, and much effort has to be spent to get around the GFW is to hire and manage a full time software team.

@Jess822 @wongr
